### ISSUE TITLE
[Bug]: Appwrite: The requested route was not found

### ISSUE THREAD

**Issue by crocofied** [coollabsio/coolify#4224]
### Error Message and Logs

If you try to install appwrite and open it now in the browser you get this error on the site:

`{"message":"The requested route was not found. Please refer to the API docs and try again.","code":404,"type":"general_route_not_found","version":"1.6.0"}`

### Steps to Reproduce

1. Add new Service
2. Select Appwrite
3. Click Deploy
4. Open the URL/Domain From Appwrite

### Coolify Version

v4.0.0-beta.364

### Are you using Coolify Cloud?

No (self-hosted)

### Operating System and Version (self-hosted)

Debian 12

**Comment by Mini-Sylar** [comment]
It seems with appwrite 1.6, a new image "appwrite-console" has been added, separate from the "appwrite" image, it's missing in the coolify template file

**Comment by Wodwes** [comment]
to install the v1.5

**Comment by Mini-Sylar** [comment]
@Wodwes Tried doing it manually, was getting a 500 error after starting the app, probably because of the strip prefix thing
Seems there's an open PR tho, #4226,

**Comment by bananacy** [comment]
lol i just discovered both coolify and appwrite, thought i'd try them and boom doesn't work :-)
{"message":"The requested route was not found. Please refer to the API docs and try again.","code":404,"type":"general_route_not_found","version":"1.6.0"}

**Comment by ilbeybey** [comment]
Yes, I have the same issue. I started to be use from 1.5.11 version until the 1.6 version is fixed. Coolify team please help us :)

**Comment by Mini-Sylar** [comment]
@ilbeybey  here's what I've gathered so far, my issues have been resolved, you can follow this PR -> #4279 

**TLDR**
- Coolify has been updated with the new 1.6.0 template but there's still some configuration you need to do manually
- If you can't update coolify here's the new template [NEW APPWRITE TEMPLATE ](https://github.com/djsisson/coolify/blob/1ee0a857533975adc6e5ba4838d621f0ee22fc7f/templates/compose/appwrite.yaml) by [djsisson](https://github.com/djsisson)
- He also updated the `ENV`, function domains work perfectly now

## Assuming you're able to start from scratch
### Traefik
- If you're on `traefik` i.e the default coolify proxy you need to disable `"strip prefix"` for `realtime` and `console` and it should work. Easiest option IMO

### Caddy 
- you need to remove the domains from both `realtime` and `console`. Also disable strip prefix (Might not matter)
- Once this is done you'll notice that in the "deployable" compose the `labels` will be gone, you'll need to add this in manually. Examples below

```yaml
appwrite-console:
  ## ... other stuff from the new template
     environment:
      - SERVICE_FQDN_APPWRITE=/console #this should already exist
    labels:
      - 'caddy_0.encode=zstd gzip'
      - 'caddy_0.handle=/console*'
      - 'caddy_0.handle.0_reverse_proxy={{upstreams}}'
      - caddy_0.header=-Server
      - 'caddy_0.try_files={path} /index.html /index.php'
      - 'caddy_0=YOUR DOMAIN ' # e.g https://appwrite.yourdomain.com or https://yourdomain.com, note the '
      - caddy_ingress_network=YOUR NETWORK   # You can find this in the deployable compose, the random string e.g ec4w4ow4coo....
```

```yaml
appwrite-realtime:
   ##... other stuff from the new template
    - '_APP_USAGE_STATS=${_APP_USAGE_STATS:-enabled}'
      - '_APP_LOGGING_CONFIG=${_APP_LOGGING_CONFIG}'
    labels:
      - 'caddy_0.encode=zstd gzip'
      - 'caddy_0.handle=/v1/realtime*'
      - 'caddy_0.handle.0_reverse_proxy={{upstreams}}'
      - caddy_0.header=-Server
      - 'caddy_0.try_files={path} /index.html /index.php'
      - 'caddy_0=YOUR DOMAIN'
      - caddy_ingress_network=YOUR NETWORK 
```
This should start work


## Data Backup
Now I know you may not be able to start from scratch so here are a few options to backup your data; 

> [!CAUTION]
> This might not be fool proof and you may lose your data, explore other options from appwrite/someone else first

#### Option 1
- Use the Appwrite CLI to pull in all your databases, functions, attributes etc.
-  Create a new service with the new changes, 
- push the changes to your new service

#### Option 2 (Did the migration but lost my data after migrating back but it's worth exploring)
- migrate to cloud 
- then migrate back after the new template is setup 

## Concerns
- You may have an issue with creating attributes, if that's the case then see [FIX](https://github.com/coollabsio/coolify/issues/4279#issuecomment-2479298124)
```
    - '_APP_WORKERS_NUM=${_APP_WORKERS_NUM:-1}'
    - '_APP_QUEUE_NAME=${_APP_QUEUE_NAME:-appwrite}'
    - '_APP_DATABASE_SHARED_TABLES=${_APP_DATABASE_SHARED_TABLES}'
```
- Strongly recommend your follow this [ISSUE](https://github.com/coollabsio/coolify/issues/4279)
- Seems like a lot of steps but it really isn't, I just typed a lot trying to simplify it as much as possible
- Regarding caddy, there's an open PR to fix the strip prefix option

**Comment by ilbeybey** [comment]
Thanks for your help @Mini-Sylar It worked!
